rotkehlchen/chain/ethereum/interfaces/ammswap/types.py

```diff
@@ -124,6 +124,10 @@ def __str__(self) -> str:
         raise RuntimeError(f'Corrupt value {self} for EventType -- Should never happen')
 
 
+UNISWAP_EVENTS_TYPES = [EventType.MINT_UNISWAP, EventType.BURN_UNISWAP]
+SUSHISWAP_EVENTS_TYPES = [EventType.MINT_SUSHISWAP, EventType.BURN_SUSHISWAP]
```

Comment:
Make those a tuple since they are faster and use less space.

```suggestion
UNISWAP_EVENTS_TYPES = (EventType.MINT_UNISWAP, EventType.BURN_UNISWAP)
SUSHISWAP_EVENTS_TYPES = (EventType.MINT_SUSHISWAP, EventType.BURN_SUSHISWAP)
```